packages/frontend/components/layout/BrowseTreeLayout/BrowseTreeItem/Teasers/Teasers.tsx

```diff
@@ -0,0 +1,66 @@
+import { graphql } from "react-relay";
+import { useMaybeFragment } from "@wdp/lib/api/hooks";
+import { formatDate } from "@wdp/lib/helpers";
+import { useTranslation } from "react-i18next";
+import { DotList } from "components/atomic";
+import { TeasersFragment$key } from "@/relay/TeasersFragment.graphql";
+
+export default function Teasers({ data }: Props) {
+  const { t } = useTranslation();
+  const entry = useMaybeFragment(fragment, data);
+
+  const entryIsSeries = entry?.schema?.identifier === "series";
+  const series = entry?.series?.pageInfo?.totalCount;
+  const papers = entry?.papers?.pageInfo?.totalCount;
+
+  return entry ? (
+    <DotList className="t-copy-lighter">
```

Comment:
I made a mistake in my design - this text should use the class `t-copy-sm` to match other metadata in lists.